wordpress-mobile/WordPress-Android
# Feature/site media image settings

This is the first of several PRs which will implement the media settings redesign. In this PR, the new settings screen has been implemented for images and videos. Full-screen preview and settings for audio & documents will be implemented in subsequent PRs (as per #6606).

Note that a design review will come after all the PRs have been merged into the feature branch. That said, feel free to comment on any obvious design issues.

To test
* Open the site media browser
* Tap an image
* Notice the snazzy new screen :)

![screenshot_1505326975](https://user-images.githubusercontent.com/3903757/30393896-7d762b66-988f-11e7-813e-c1745fd7a4ba.png)
